addons/mail/controllers/thread.py

```diff
@@ -105,6 +105,8 @@ def mail_message_post(self, thread_model, thread_id, post_data, context=None, **
                 kwargs["partner_emails"], kwargs.get("partner_additional_values", {})
             )]
         post_data["partner_ids"] = list(set((post_data.get("partner_ids", [])) + new_partners))
+        if "everyone" in special_mentions:
```

Comment:
this seems weird to me.... I would prefer to have a method handling all mentions specially on the server side rather than having a string case check in a general message post. The way I suggest would be to check if it is a special mention on the client side and directly call the special all mentions handler. Not sure how @alexkuhn think of this idea.

Reply:
and as seb said, a special handler might be better to handle the public case and more complex case

Reply:
I also agree that special is one of those terms that should not be used (just like nice, awesome, etc...).
Yes the "ever" was intentional as it is part of the specification [task-2366705](https://www.odoo.com/web#id=2366705&cids=1&menu_id=4720&action=333&active_id=1519&model=project.task&view_type=form) (though on Discord it appears immediately).
I agree that `@everyone` is more like a command than a mention but the tricky part is that it must appear on the same list as the other partner mentions (because of the delimiter "@").

Reply:
@zel-odoo The client-side is not necessarily aware of all channel members, so it's best for it being handled server-side.

Reply:
Do we need a list of special mention? We plan to support here or something in the future?
Otherwise just everyone=False/True as param would be sufficient

Reply:
I think that working on a list of special mentions does not add much more complexity than handling just one case. Also it would minimize the diff in case other special mentions need to be added.